Documentation for PDB support for custom resource with scale subresource (#14519)
kubernetes/website: content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

5
```

> Custom resources with the scale subresource enabled can also be protected with [PodDisruptionBudgets](docs/tasks/run-application/configure-pdb/).

Reviewer: ```suggestion
You can use a [PodDisruptionBudget](docs/tasks/run-application/configure-pdb/) to protect custom resources that have the scale subresource enabled.
```
